api/src/pcapi/routes/backoffice/finance/blueprint.py

```diff
@@ -452,9 +452,15 @@ def get_finance_incident_validation_form(finance_incident_id: int) -> utils.Back
     finance_incident = (
         finance_models.FinanceIncident.query.filter_by(id=finance_incident_id)
         .options(
-            sa.orm.joinedload(finance_models.FinanceIncident.booking_finance_incidents).load_only(
-                finance_models.BookingFinanceIncident.newTotalAmount
-            ),
+            sa.orm.joinedload(finance_models.FinanceIncident.booking_finance_incidents)
+            .load_only(finance_models.BookingFinanceIncident.newTotalAmount)
+            .joinedload(finance_models.BookingFinanceIncident.collectiveBooking)
```

Comment:
Ce joinedload charge probablement tous les champs de `collectiveBooking`. Comme tu as mis des `load_only` par ailleurs, tu ne le souhaitais probablement pas.
```suggestion
            .joinedload(finance_models.BookingFinanceIncident.collectiveBooking)
            .load_only(educational_models.CollectiveBooking.id)
```